nixos/modules/services/networking/wireguard.nix

```diff
@@ -137,12 +137,12 @@ let
   generateConf = name: values: pkgs.writeText "wireguard-${name}.conf" ''
```

Comment:
This module should be probably rewritten to use wg-quick, since it lakes of features like auto-mtu and the default gateway over wireguard trick.